services-custom/dynamodb-enhanced/src/functionalTests/BasicCrudTest.java

```diff
@@ -0,0 +1,453 @@
+/*
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package functionalTests;
```

Comment:
hmm, we don't currently have a `functionalTests` folder, how is it different from `test` or `it`?

Should we move it under `package software.amazon.awssdk.services.dynamodb.xxx`

Reply:
Will fix.